src/routes/notes/_.$id.edit.lazy.tsx

```diff
@@ -0,0 +1,15 @@
+import { createLazyFileRoute, Link } from "@tanstack/react-router";
+
+export const Route = createLazyFileRoute("/notes//$id/edit")({
```

Comment:
This URL may look fucked, but it's not. This is auto generated, and displays as expected in the app (/notes/:id/edit).

I will figure out a way to have it be prettier later, right now it's only aesthetic and works perfectly fine.